api/types/plugin.go

```diff
@@ -143,6 +142,17 @@ type PluginConfigNetwork struct {
 	Type string `json:"Type"`
 }
 
+// PluginConfigRootfs plugin config rootfs
+// swagger:model PluginConfigRootfs
+type PluginConfigRootfs struct {
+
+	// diff ids
+	DiffIds []string `json:"diff_ids"`
```

Comment:
Discussed with @dnephin and this should be solvable with `x-go-name:` but it has some issue atm. The json names are correct so we won't have breaking changes when we switch the go names.